framework/src/main/java/org/checkerframework/framework/util/ContractsUtils.java

```diff
@@ -473,7 +498,6 @@ private AnnotationMirror getAnnotationMirrorOfMetaAnnotation(
                 result.addAll(getConditionalPostcondition(a));
             }
         }
-
```

Comment:
Why did you delete this line?

Reply:
I run `./gradlew reformat`after every change. Maybe that caused this change.

Reply:
That is unlikely.  The formatting routines do not remove whitespace between two lines of code.